features/product/viewing_products/viewing_a_products_lowest_price_from_30_days_before_the_discount.feature

```diff
@@ -0,0 +1,76 @@
+        And this product's price was discounted to "$21.00"
+        When I check this product's details
+        Then I should see "$21.00" as this product's lowest price from 30 days before the discount
+
+    @todo
+    Scenario: Seeing the lowest price information on a product that has a discount and then it has a less attractive discount change
+        Given the store has a product "Wyborowa Vodka" priced at "$37.00"
+        And this product's price was discounted to "$21.00"
+        And this product's discounted price changed to "$30.00"
+        When I check this product's details
+        Then I should see "$21.00" as this product's lowest price from 30 days before the discount
+
+    @todo
+    Scenario: Seeing the lowest price information on a product that has a discount and then it has a more attractive discount change
+        Given the store has a product "Wyborowa Vodka" priced at "$37.00"
+        And this product's price was discounted to "$21.00"
+        And this product's discounted price changed to "$9.00"
+        When I check this product's details
+        Then I should see "$9.00" as this product's lowest price from 30 days before the discount
+
+    @todo
+    Scenario: Seeing the lowest price information on a product that has a discount, over a month passes, and then it has a less attractive discount change
+        Given it is "2023-03-14" now
+        And the store has a product "Wyborowa Vodka" priced at "$42.00"
+        And this product's price was discounted to "$21.00"
+        And it is "2023-04-15" now
+        And this product's discounted price changed to "$33.00"
+        When I check this product's details
+        Then I should see "$33.00" as this product's lowest price from 30 days before the discount
+
+    @todo
+    Scenario: Seeing the lowest price information on a product that has a discount, over a month passes, and then it has a more attractive discount change
+        Given it is "2023-03-14" now
+        And the store has a product "Wyborowa Vodka" priced at "$42.00"
+        And this product's price was discounted to "$21.00"
+        And it is "2023-04-15" now
+        And this product's discounted price changed to "$19.00"
+        When I check this product's details
+        Then I should see "$19.00" as this product's lowest price from 30 days before the discount
+
+    @todo
+    Scenario: Not seeing the lowest price information on a product that had a discount and the discount was removed
+        Given the store has a product "Wyborowa Vodka" priced at "$37.00"
+        And this product's price was discounted to "$21.00"
+        And this product's discount was removed
+        When I check this product's details
+        Then I should not see information about its lowest price
+
+    @todo
+    Scenario: Not seeing the lowest price information on a product that had a discount, over a month passes, and then the discount was removed
```

Comment:
Rather than checking if we see the lowest price after the timeframe has passed, this should check whether the info is there when the discount was added 15 days ago etc., and then got removed, but that's pretty much covered in the scenario above.

Reply:
I removed it as indeed this test wasn't valuable